roles/openshift_prometheus/tasks/oc_apply.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Checking generation of {{file_content.kind}} {{file_content.metadata.name}}
```

Comment:
@ewolinetz As far as I can tell from openshift logging project components, there are now oc modules for:
serviceaccount, secret, service & configmap.
Other objects (route, deployment, rolebinding) should be created using oc_obj module instead of running "oc apply" with command module. is that correct?